lib/src/utils/smooth_video_progress.dart

```diff
@@ -0,0 +1,91 @@
+library smooth_video_progress;
```

Comment:
Please leave a comment for credit to this package
`// Package credits : https://pub.dev/packages/smooth_video_progress`

Reply:
Looks good. Forgot to add credits to the original package.